app/views/main/home.html.erb

```diff
@@ -32,11 +32,6 @@
     </div>
   </div>
 </div><!-- end yield -->
-<div class="github-fork-ribbon-wrapper right-bottom">
-  <div class="github-fork-ribbon">
-    <a href="https://github.com/metamaps/metamaps" target="_blank">Fork me on GitHub</a>
-  </div>
-</div>           
```

Comment:
Not that I'm opposed to removing, but what was your reasoning? Is it because it makes it wonky when the screen is small?